client/app/models/contact.coffee

```diff
@@ -66,13 +66,16 @@ module.exports = class Contact extends Backbone.Model
                         @fetch()
 
                     error: options.error
+        super
+
 
+    validate: (model, options) ->
         # Handle specific attributes.
-        attrs = model.toJSON()
+        attrs = _.clone(model)
```

Comment:
Since `attrs` passed as arguments are already a JSON export, you can skip the `clone` call, no?